tests/profiles/fontval_test.py

```diff
@@ -1,11 +1,13 @@
 import pytest
+import shutil
 
 from fontbakery.codetesting import (TEST_FILE,
                                     assert_results_contain)
 from fontbakery.checkrunner import ERROR
 from fontbakery.profiles import fontval as fontval_profile
 
-
+@pytest.mark.skipif(not shutil.which("FontValidator"),
+  reason="FontValidator is not installed on your system")
```

Comment:
This will reduce the code-testing coverage depending on the system setup on the computer where the code-tests are running.

I can see how this can be helpful to facilitate setting up a development environment without the hassle of installing MSFontVal locally. Unfortunately, it also means that if a developer ends up breaking the MSFontVal related code in fontbakery, he or she may only notice the problem after sending a pull request because we do explicitly install MSFontVal in our continuous integration setup (currently running on GHActions)

In other words, since we install MSFontVal on GHActions builds, this will not really reduce the test coverage, but will only delay the detection of any problem with fontval. So it may actually be OK to use this `skipif`.